### ISSUE TITLE
Both deebots not responding, only after reboot of pimatic

### ISSUE THREAD

**Issue by djmvt** [bertreb/pimatic-deebot#11]
As stated in the title, both my deebots (ozmo 900 and ozmo 950) are no longer responding with the Pimatic plugin. Only after a reboot the deebots are working again.
Status of the device is online but doesn't update on manual start (or via the ecovacs app)

Anything I can do to fix this?

**Comment by bertreb** [comment]
The plugin is updated to 0.0.35. I updated the used ecovacs lib to version 0.6.x
That could solve the issue.

**Comment by djmvt** [comment]
Installed it (gave a few errors during install, also some 404 errors but went through without further issues) but now both robots won't connect and give an error in the log: cannot find module "duplexify".

**Comment by bertreb** [comment]
You can install duplexify manually in the directory pimatic-deebot (npm install duplexify)

**Comment by djmvt** [comment]
Looks like it killed Pimatic. Yes I installed it in the right dir but after that the Pimatic GUI didn't start anymore. I've deleted the pimatic-deebot plugin in the config.json to see if I could get it back but the GUI no longer works.

**Comment by bertreb** [comment]
If you start pimatic in commandline mode manually, what does the log say?

**Comment by djmvt** [comment]
The log gave a lot of errors, all related to problems with pimatic-deebot. I've deleted the entire Pimatic folder and went back to a Pimatic backup I've made in January. This one booted without issues (except deebot staying offline). Tried to update the deebot plugin again, same issue: Pimatic GUI no longer starting, loads of errors. 
So did the same thing again by putting back my backup. I'm going to delete the deebot plugin, it generates a lot of issues on my system. Don't know why and I'm pretty sure it isn't the plugin by itself. I've had issues before with canvas, it just doesn't seem compatible with my setup and I'm pretty sure canvas or things that have to do with canvas do kill my Pimatic system. 
As with other plugins: it's not necessary to have them and I rather have a Pimatic system with less plugins running without issues than having plugins installed that could break the system.
